front-end/src/components/VotingHistoryTable.js

```diff
@@ -124,8 +159,32 @@ function Table({ data, articleFilters, reelectionToggle, articles }) {
           })}
         </tbody>
       </table>
+      <div id="pagination-controls" className="d-flex justify-content-center align-items-center ml-0 mt-3 mb-3 mr-0">
+        <button
```

Comment:
I think two non-critical suggestions for this pagination:

1. Should we hide it entirely when there's just a single page? ie when no pagination is possible?
2. On mobile, should we put the pagination above the table when we display it?

Reply:
1) I would be happy to try rendering this conditionally when pagination is needed. 
2) I'm not sure I have strong intuitions here. What's driving your thought process? I'd be willing to explore it, but don't know of what would be preferable one way or another honestly.

Reply:
@GrahamGoudeau bump